### DISCUSSION TITLE
Prediction of properties at different temperatures

### DISCUSSION THREAD

**Discussion by wthd-hhp** [chemprop/chemprop#1113] [Q&A] [answered]
I would like to ask if I want to predict a certain property of a substance at different temperatures, can I add features with different temperatures when training a model.

**Comment by JacksonBurns** [comment, marked as answer]
Hi @wthd-hhp they way this is usually done is by passing the temperature in as an extra graph-level feature to be embedded to the learned representation. You can do this with the `--descriptors-path` argument - see the docs for more info.

Here's one example paper where this is done for solubility prediction at varying temperature: https://doi.org/10.26434/chemrxiv-2024-93qp3
and here's the associated code, which shows how to do this via a Python script: https://github.com/JacksonBurns/fastsolv/blob/main/paper/models/chemprop/train.py
